### PR TITLE
Add missing i18n to "Latest Posts" block settings strings

### LINKED ISSUE #19031: [i18n] Block "Latest Posts" strings missing i18n.
**Describe the bug**
The block "Latest Posts" has a few strings missing i18n.

**To reproduce**
1. Edit a post
2. Add Latest Posts block
3. On the sidebar, activate "Post Content Settings"
4. The strings "Show:", "Excerpt" and "Full Post" aren't translatable.

**Screenshots**

Reported in core ticket https://core.trac.wordpress.org/ticket/48926

### PR DESCRIPTION
**Pull request by pedro-mendonca**
Fixes #19031 

## Description
Issue described in #19031

### CODE REVIEW

**Review by youknowriad — APPROVED** [review]
Good catch, thanks for the fix.

File: packages/block-library/src/latest-posts/edit.js [hunk 1]
```diff
@@ -82,11 +82,11 @@ class LatestPostsEdit extends Component {
 					/>
 					{ displayPostContent &&
 					<RadioControl
-						label="Show:"
+						label= __( 'Show:' )
```
Comment (aduth) [inline packages/block-library/src/latest-posts/edit.js]: This is a syntax error.

```suggestion
						label={ __( 'Show:' ) }
```

**Review by aduth — CHANGES_REQUESTED** [review]

File: packages/block-library/src/latest-posts/edit.js (hunk 1 above)
Comment (youknowriad) [reply packages/block-library/src/latest-posts/edit.js]: how did I miss this :)

Comment (pedro-mendonca) [reply packages/block-library/src/latest-posts/edit.js]: Fixed :)

**Review by aduth — APPROVED** [review]

**Pull request merged**